### DISCUSSION TITLE
Is Ruffle lighter than the Flash Player itself??

### DISCUSSION THREAD

**Discussion by yumnycheesecakes** [ruffle-rs/ruffle#12041] [General]
Hi everyone, is Ruffle lighter than the Flash Player itself?? I tried Flash Player Standalone on my X11 Gnome Ubuntu, and it had some bugs. But Ruffle didn't had bugs. I know that they run on different libraries, but is Ruffle lighter??

**Comment by yumnycheesecakes** [comment]
In case anyone wonders, I meant "lighter" is "more lightweight"

**Comment by Lord-McSweeney** [comment]
What bugs specifically?

**Comment by yumnycheesecakes** [comment]
> What bugs specifically?

like graphics bugs, i can't tell them exactly

**Comment by Lord-McSweeney** [comment]
If you don't have a specific bug, I can't really help. Note that it's possible that FP is using some older graphics drivers which are not as supported on your system, while Ruffle is using newer, more supported ones. In terms of "lightweight"-ness, though, FP is definitely more lightweight. It uses less memory and is almost always more performant.
